### ISSUE TITLE
Additional plugins

### ISSUE THREAD

**Issue by avelino** [editor-bootstrap/vim-bootstrap#101]
The configuration of .vimrc varies a lot of taste, we can think of an area in the web interface (called Advanced) for the advanced user select which plugins he wants to put as additional, so generate the .vimrc with everything that the user like.

Vim template of each technology (language) comes with the basics you need to use that language. We can follow this path to the template of each linguaem here, suggest which plugins will be used and advanced user clears the that does not like.

What do you think?

Solution for:
fixed: #100
fixed: #98
fixed: #96
fixed: #92
fixed: #80

...

**Comment by gilsondev** [comment]
I agree. Advanced section to frameworks and other tools (ex.: Markdown, etc)

**Comment by avelino** [comment]
Today exist vimawesome.com and this API:
http://vimawesome.com/api/plugins?query=tag%3Apython&page=1

**Comment by gilsondev** [comment]
This can be a nice integration.

**Comment by avelino** [comment]
Hey guys, check implementation:
https://github.com/avelino/vim-bootstrap/tree/selected_plugin
http://5.vim-bootstrap.appspot.com/ (not yet in production)

Now we need a help of a frontend! @hersonls

**Comment by hersonls** [comment]
@avelino great, I will make a frontend for this feature

**Comment by gilsondev** [comment]
Nice ;)

**Comment by FireAntJoe** [comment]
Any word on this feature?

**Comment by avelino** [comment]
@FireAntJoe We need collaborator to implement

**Comment by FireAntJoe** [comment]
@avelino okay

**Comment by urienix** [comment]
Greetings guys, I only come to say, it would be great if you added VIM Icons inside the Plugins package

You can find it at: https://github.com/ryanoasis/vim-devicons

I tried it and I loved it

![image](https://user-images.githubusercontent.com/20466553/125981092-64e37697-2787-4941-a240-22c90ede282d.png)
